### PR TITLE
Remove commitizon and update pipeline

### PR DESCRIPTION
**Pull request by jingyi-gao-ttd**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -62,10 +62,26 @@ All new source files _must_ be in TypeScript, with useful type information for a
...
+   ```bash
+   npm install
+   ```
+2. Use `Publish SDK to NPM and CDN` Action.
+
+### Prerelease
+
+If you intend to publish a prerelease version:
+
+1. Update the version number in package.json with the desired prerelease tag. For instance: `3.0.3-alpha.0`
```
Comment (mcollins-ttd) [inline readme.md:82]: And then run `npm install`?

File: .github/workflows/publish-package-to-npmjs.yml [hunk 2]
```diff
@@ -29,8 +19,6 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-          cache: "npm"
-          cache-dependency-path: ${{ env.WORKING_DIR }}/package-lock.json
```
Comment (mcollins-ttd) [inline .github/workflows/publish-package-to-npmjs.yml:33]: Why were these lines removed?

File: readme.md (hunk 1 above)
Comment (jingyi-gao-ttd) [reply readme.md:82]: `npm install` will update the `package-lock.json`, I think we should better not manually update it

File: .github/workflows/publish-package-to-npmjs.yml (hunk 2 above)
Comment (jingyi-gao-ttd) [reply .github/workflows/publish-package-to-npmjs.yml:33]: This build is running before package publish now, thus, there will not be a cache for it

**Review by alex-yau-ttd — APPROVED** [review]
